packages/application-shell/src/test-utils/test-utils.tsx

```diff
@@ -220,12 +223,12 @@ const wrapIfNeeded = (
 type TApolloProviderWrapperProps = {
   apolloClient?: ApolloClient<NormalizedCacheObject>;
   mocks: ReadonlyArray<MockedResponse>;
-  disableApolloMocks: boolean;
+  enableApolloMocks: boolean;
```

Comment:
I thought this was called `useApolloMocks`. At lease the change log stated so.

Reply:
Ah thanks, it should be `enableApolloMocks`.

I find `useApolloMocks` to be confusing with some kind of react hook...

Reply:
Yeap, I also disable the `use*` toggle naming.